### DISCUSSION TITLE
rename default branch name to main

### DISCUSSION THREAD

**Discussion by KiaraGrouwstra** [srid/haskell-template#118] [Ideas]
this would put the project in line with current trends

**Comment by srid** [comment]
Does clicking on the "Use this template" button on GitHub not already automatically create the new repository with the user's default branch?

**Reply by KiaraGrouwstra** [reply]
@srid for me it turns out it still defaulted to master branch, despite my [default branch](https://github.com/settings/repositories) being set to main.
